index.js

```diff
@@ -232,6 +243,446 @@
         }
     }
 
+    /*
+        Sets Default Torrent Trackers
+        the GUID is the torrent ID
+     */
+    function addTrackers(guid, callback) {
+        console.log("Adding Default Trackers");
+        post({
+                "method": "core.set_torrent_trackers",
+                "params": [guid,
+                    [{"tier": 0, "url": "http://tracker.trackerfix.com:80/announce"}, {
```

Comment:
Seems weird to have these hardcoded into the repo, I would source them from a config file of some kind, and allow this method to take any number of trackers in an array. That way the functionality can be used again, and you can have a default list.